tools/cpp/runfiles/runfiles_src.cc

```diff
@@ -103,6 +103,12 @@ bool PathsFrom(const std::string& argv0, std::string runfiles_manifest_file,
                std::function<bool(const std::string&)> is_runfiles_directory,
                std::string* out_manifest, std::string* out_directory);
 
+bool PathsFrom(std::string runfiles_manifest_file,
```

Comment:
Can you add some comments to explain what the PathsFrom functions do? I took me some time to parse them. I know we probably should have added them for the existing ones, but it's never too late to improve the readability ;)